src/rsa/signing.rs

```diff
@@ -219,6 +217,20 @@ impl RSAKeyPair {
     pub fn public_modulus_len(&self) -> usize {
         self.n_bits.as_usize_bytes_rounded_up()
     }
+
+    /// Exports the public modulus as big endian to the output array.
+    ///
+    /// The array should be of at least the size returned by
+    /// `public_modulus_len`.
+    pub fn export_public_modulus(&self, out: &mut [u8])
+                    -> Result<(), error::Unspecified> {
+        self.n.fill_be_bytes(out)
```

Comment:
Here's an excerpt from the documentation for `verify_rsa`:
```rust
/// `n` is the public key modulus and `e` is the public key exponent. Both are
/// interpreted as unsigned big-endian encoded values. Both must be positive
/// and neither may have any leading zeros.
```

These requirements come directly from JWA https://tools.ietf.org/html/rfc7518#section-6.3.1.1 (for n):
> Note that implementers have found that some cryptographic libraries prefix an extra zero-valued octet to the modulus representations they return, for instance, returning 257 octets for a 2048-bit key, rather than 256.  Implementations using such libraries will need to take care to omit the extra octet from the base64url-encoded representation.

and (for e):

> For instance, when representing the value 65537, the octet sequence to be base64url-encoded MUST consist of the three octets [1, 0, 1];

Thus supporting extra zero padding doesn't do anybody any good, AFAICT.